.env.example

```diff
@@ -1,6 +1,7 @@
 ALCHEMY_API_KEY=
 COINMARKETCAP_API_KEY=
 ETHERSCAN_API_KEY=
+AVASCAN_API_KEY=
```

Comment:
Please use Snowtrace like a normal person

Reply:
Ahahah I'm just too used to the old name

Reply:
Please change the name then